### ISSUE TITLE
Ripple directive does not work 

### ISSUE THREAD

**Issue by maximbelo** [primefaces/primevue-tailwind#213]
Ripple directive does not work with the Tailwind setup.

**Comment by timyourivh** [comment]
For me it did, but after making changes it stopped working for an yet unknown reason. Even after reverting my repo to the moment it did work. It's unclear to me what is going on. Here are the things I tried to bring it back, but without success:

- Restart dev server
- Using a different browser
- Clearing Vite cache
- Deleting node_modules folder and re-installing
- Building
- Restarting PC

It **does** show in the html, but it's not visible for some reason:

![ripple effect demo](https://github.com/primefaces/primevue-tailwind/assets/60601502/7cd07fe2-374e-4241-be12-045f2081a12a)
*Note that I use a fade effect when active, the ripple is not present*

<details>
  <summary>Source of example above</summary>
  
  ```vue
  <div class="w-40">
    <div v-ripple class="bg-primary-500 active:bg-primary-400 dark:bg-primary-400 dark:active:bg-primary-300 text-surface-0 dark:text-surface-900 flex select-none justify-center items-center shadow-md rounded-md p-6 font-bold transition-colors ease-in-out duration-200">Default</div>
  </div>
  ```
</details>


<details>
  <summary>Source of prime vue plugin</summary>
  
  ```js
  import PrimeVue from 'primevue/config';
  import Ripple from 'primevue/ripple';
  import defaultTheme from '../themes/defaultTheme';

  export const primeVuePlugin = {
      install(app) {
          app.use(PrimeVue, {
              ripple: true,
              unstyled: true,
              pt: defaultTheme
          });

          app.directive('ripple', Ripple)
      }
  }
  ```
</details>

Adding a [custom](https://tailwind.primevue.org/ripple/#custom) example results in this:

![ripple effect demo 2](https://github.com/primefaces/primevue-tailwind/assets/60601502/06eb5cfe-39c2-4844-bb95-d241787bbdd2)

It's a real head-scratcher for me.

**Comment by timyourivh** [comment]
I found the problem!
It basically comes down to the colors in tailwind. In `wind/ripple/index.js` the color `bg-surface-0/50` is used.
In my case I modified the color variables in my app.css:

Example:
```css
:root {
    /* ... */
    --surface-50: 248, 249, 250;
    /* ... */
}
```

**Notice the comma's**, I didn't think it would make a difference if I would leave them in since it worked, but after removing them the effect was restored. So to fix it I did:

```diff
-    --surface-50: 248, 249, 250;
+    --surface-50: 248 249 250;
```

for all my colors.

### TL;DR:
The tailwind color being used for the ripple effect does not exist, not correctly defined or is not compatible to make transparent.

**Comment by maximbelo** [comment]
Ah, yes, good catch! It's the commas that are causing the issue.
